nservicebus/upgrades/asp-6to1.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Azure Storage Persistence Upgrade Version 6 to 1
```

Comment:
Some possibilities:

- Migrate from NServiceBus Azure Version 6 to NServiceBus Azure Storage Persistence Version 1
- Migrating to Azure Storage Persistence Version 1